environment.conf.json

```diff
@@ -25,10 +25,21 @@
     "title": "Signalen Informatievoorziening Amsterdam",
     "shortTitle": "SIA",
     "siteTitle": "SIA - Signalen Informatievoorziening Amsterdam",
+    "headerTitle": "",
+    "smallHeaderTitle": "SIA",
     "footer1": "Bel het Gemeentelijk informatienummer: 14 020",
-    "footer2": "op werkdagen van 08.00 tot 18.00 uur."
+    "footer2": "op werkdagen van 08.00 tot 18.00 uur.",
+    "incidentForm": {
+      "step5": {
+        "consent": "Ja, ik geef de gemeente Amsterdam toestemming om mijn contactgegevens te delen met andere organisaties, als dat nodig is om mijn melding goed op te lossen."
+      },
+      "step7": {
+        "extraInfo": "Voor spoedeisende zaken kunt u ook telefonisch contact opnemen met 14 020."
+      }
+    }
```

Comment:
I'm not sure about the explicit `step5` and `step7` prop names. What if there is steps added in between or the order of steps is changed? Then the numbering doesn't make sense any more. I think we're better off choosing proper prop names instead of trying to nest it in a clever way. I propose using `allowConsent` and `urgentContactInfo` in these cases.

That having said, these two Amsterdam specific texts aren't the only two that refer to the municipality of Amsterdam. There are texts in the definitions of `overlast-van-dieren`, `overlast-op-het-water`, `overlast-van-en-door-personen-of-groepen`, `wegen-verkeer-straatmeubilair` and `wizard-step-7-fout` that can only be applied to Amsterdam.

Reply:
I agree that these explicit references to the steps aren't very pretty. I'll update those, but keep them nested in `incidentForm`. Or do you prefer not to have any nesting at all?

All the other references you mention are the additional questions and those are being moved to the backend anyway. The only one left is the one you mention in step-7, but I moved that one to the configuration as well already, as you can see.

Reply:
With the names you came up with, I'd say it doesn't need to be nested at all actually!

I'd use a different name for `allowConsent` though. Like this, it seems like a pleonasm. A better description would be `consentToContactSharing`.